framework/src/packets/raw.rs

```diff
@@ -1,17 +1,45 @@
+use std::cmp;
 use common::Result;
 use native::zcsi::{mbuf_alloc, MBuf};
 use packets::{buffer, Header, Packet};
+use packets::buffer::read_slice;
 
 /// Unit header
 impl Header for () {}
 
 /// The raw network packet
 ///
 /// Simply a wrapper around the underlying buffer with packet semantic
+#[derive(Debug)]
 pub struct RawPacket {
     mbuf: *mut MBuf,
 }
 
+/// Compare RawPackets. This probably isn't something you want to be doing a lot of at runtime.
```

Comment:
Triple `/` is for rustdoc. General code comment (I think this is?) use double `/`.